build/params_2k.go

```diff
@@ -67,9 +67,9 @@ var UpgradeThunderHeight = abi.ChainEpoch(-23)
 
 var UpgradeWatermelonHeight = abi.ChainEpoch(-24)
 
-var UpgradePineappleHeight = abi.ChainEpoch(20)
+var UpgradeDragonHeight = abi.ChainEpoch(20)
 
-var UpgradeMangoHeight = UpgradePineappleHeight + 10
+var UpgradeMangoHeight = UpgradeDragonHeight + 100
```

Comment:
- Knight
- Serpent
- Slayer
- ...

Pinapple -> mango, dragon -> mango???? no.

Reply:
Lollol, I hear Dragons like Mango??! Updated Mango to Phoenix in: https://github.com/filecoin-project/lotus/pull/11599/commits/cea5ae65af719c905dd4ffe629b5ebfad1962815